storage/rocksdb/rdb_datadic.cc

```diff
@@ -1413,17 +1413,22 @@ int Rdb_key_def::unpack_record(TABLE *const table, uchar *const buf,
     return HA_ERR_ROCKSDB_CORRUPT_DATA;
   }
 
-  // For secondary keys, we expect the value field to contain unpack data and
-  // checksum data in that order. One or both can be missing, but they cannot
-  // be reordered.
+  // For secondary keys, we expect the value field to contain unpack data,
```

Comment:
@georgelorchpercona What I meant was:

For secondary keys, we expect the value field to contain index flags, unpack data,
and checksum data in that order. One or all can be missing,
but they cannot be reordered.